src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -71,6 +71,7 @@ def load_controllers(mgrmodule):
 
 
 def _json_error_page(status, message, traceback, version):
+    cherrypy.serving.response.headers['Content-Type'] = 'application/json'
```

Comment:
Will be superseded by #50. I also don't know why we need the `serving` in there (as it works well without it).

Reply:
cherrypy.request and cherrypy.response are redirected to cherrypy.serving, so it does not matter if you use it or not. But i think we should prefer NOT to use cherrypy.serving.xxx.